src/pages/signin/LoginForm/BaseLoginForm.tsx

```diff
@@ -60,7 +59,7 @@ function BaseLoginForm({account, credentials, closeAccount, blurOnSubmit = false
     const {translate} = useLocalize();
     const input = useRef<BaseTextInputRef | null>(null);
     const [login, setLogin] = useState(() => Str.removeSMSDomain(credentials?.login ?? ''));
-    const [formError, setFormError] = useState<TranslationPaths | undefined>();
```

Comment:
We had related issue when form error was shown and user changed language in login page the error would still be in old language. We basically applied old logic to this page reverting its changes. Unfortunately I didn't see this PR before our implementation, I would've discussed this more thoroughly. But so far our changes works as expected

Related issue: https://github.com/Expensify/App/issues/44849